firmware/common/portapack_persistent_memory.cpp

```diff
@@ -240,6 +240,9 @@ struct data_t {
     // Daylight savings time
     dst_config_t dst_config;
 
+    // fake brightness level (not switch, switch is in another place)
+    uint16_t fake_brightness_level : 4;
```

Comment:
Since pmem is a very limited resource, I suggest that you take some of the unused or placeholder bits above, versus adding a new pmem field.  (Any new pmem fields that are added should be padded out to 32 bits anyway by defining additional unused bits, so that we maintain 32-bit alignment for future larger fields that could be added.)

Reply:
Thanks! I moved to use the unused one under encoder sensitive level